metadata.json

```
@@ -14,19 +14,19 @@
     },
     {
       "name": "puppet-telegraf",
-      "version_requirement": ">= 5.0.0 < 6.0.0"
+      "version_requirement": ">= 5.0.0 < 7.0.0"
```

There's already a 7.1 release of the module
```suggestion
      "version_requirement": ">= 5.0.0 < 8.0.0"
```

---

fixed